### ISSUE TITLE
When using --help option there is a missing "\n" at the end of the help page

### ISSUE THREAD

**Issue by ricardojfc** [validator/validator#2065]
### Summary

When using --help option there is a missing "\n" at the end of the help page

For reproducing the bug in linux, run:
```
./vnu-runtime-image/bin/vnu --help
```
The last line doesn't have a `\n`:
```
👉 26.4.2 (8511a6e) changelog: https://github.com/validator/validator/commits/8511a6e
```

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
n/a
```

### Checker error messages

```shell
n/a
```

### Steps to reproduce

1. Run `./vnu-runtime-image/bin/vnu --help`

### Expected behavior

The last line of the help page should have a `\n`

### Actual behavior

The last line of the help page doesn't have a `\n` so the prompt is shown after the las line character

**Comment by j9t** [comment]
Thanks for reporting this, @ricardojfc!
